src/calculateRentalCost.js

```diff
@@ -5,6 +5,21 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const price = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    return price * days;
+  }
+
+  if (days >= 3 && days < 7) {
+    return price * days - smallDiscount;
+  }
+
+  if (days >= 7) {
+    return price * days - bigDiscount;
+  }
```

Comment:
The conditions in the `if` statements use magic numbers. It would be better to define constants that explain what these numbers represent, as per the checklist. For instance, `const MIN_DAYS_FOR_SMALL_DISCOUNT = 3;` and `const MIN_DAYS_FOR_BIG_DISCOUNT = 7;` would make the code more understandable.